app/modules/intelligence/agents/chat_agents/pydantic_agent.py

```
@@ -0,0 +1,249 @@
+        )
+
+    def _create_task_description(
+        self,
+        task_config: TaskConfig,
+        ctx: ChatContext,
+    ) -> str:
+        """Create a task description from task configuration"""
+        if ctx.node_ids is None:
+            ctx.node_ids = []
+        if isinstance(ctx.node_ids, str):
+            node_ids = [ctx.node_ids]
```

Ruff (0.8.2)

Local variable `node_ids` is assigned to but never used

Remove assignment to unused variable `node_ids`

(F841)